### ISSUE TITLE
[BUG] AAS SchemaLocation must not have blank in schema address

### ISSUE THREAD

**Issue by bs-jokri** [eclipse-esmf/esmf-sdk#461]
**Describe the bug**
the XML attribute schemaLocation of the generated AAS document in XML format does contain a blank
```
xsi:schemaLocation="https://admin-shell.io/aas/3/0 AAS.xsd"
```
The blank should be removed.

Quite likely this problem comes from the used AAS4j library, for which a bug ticket was already created
https://github.com/eclipse-aas4j/aas4j/issues/186

A quick fix for this problem is introduced in PR 
https://github.com/eclipse-esmf/esmf-sdk/pull/459
as soon as a fix is available in AAS4j, the fix must be replaced with an updated library version.

**Where**
AAS XML document generator

**Comment by bs-jokri** [comment]
fixed by #459

**Comment by chris-volk** [comment]
#459 will fix this, can afterwards be closed.

**Comment by bs-jokri** [comment]
Test from the AAS4J team identified a different root cause for that problem (https://github.com/eclipse-aas4j/aas4j/issues/186). Thus the fix introduced in #459 should be removed again. Tested it and worked
